packages/client/wallets/smart-wallet/package.json

```diff
@@ -22,11 +22,7 @@
         "build": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --minify --dts --sourcemap",
         "build-no-minify": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --dts --sourcemap",
         "dev": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --dts --sourcemap --watch",
-        "link:common-sdk-base": "yalc add @crossmint/common-sdk-base && yalc link @crossmint/common-sdk-base && pnpm i && pnpm build && yalc push",
-        "test": "cross-env NODE_ENV=test jest",
-        "test-coverage": "cross-env NODE_ENV=test jest --coverage",
-        "test:starter": "pnpm run build && pnpm --filter @crossmint/client-sdk-smart-wallet-react-starter exec playwright test",
-        "unlink:common-sdk-base": "yalc remove @crossmint/common-sdk-base && pnpm i"
+        "test": "vitest run"
     },
```

Comment:
Linking is now done via workspace imports, we didn't use test coverage and `test:starter` isn't ready atm.